### PR TITLE
Update comments label in widget

### PR DESCRIPTION
**Pull request by lisasy**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [X] Optimization
- [ ] Documentation Update

## Description
In the widgets container, update the "comments" string so that the format is `3 comments` instead of `Comments: 3`.

This format uses more human-friendly language.

## QA Instructions, Screenshots, Recordings

Screen Shot 2020-10-30 at 1 07 10 PM
Screen Shot 2020-10-30 at 1 07 27 PM
Screen Shot 2020-10-30 at 1 07 42 PM

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] Yes
- [x] No, and this is why: it doesn't seem necessary
- [ ] I need help with writing tests

## Added to documentation?

- [ ] Docs.forem.com
- [ ] README
- [ ] No documentation needed

### CODE REVIEW

File: app/views/articles/_widget_list_item.html.erb [hunk 1]
```diff
@@ -2,8 +2,10 @@
   <%= plucked_article[1] %>
   <% if show_comment_count %>
     <div class="crayons-link__secondary">
-      <% if plucked_article[2] > 0 %>
-        Comments: <%= plucked_article[2] %>
+      <% if plucked_article[2] == 1 %>
+        <%= plucked_article[2] %> comment
+      <% elsif plucked_article[2] > 1 %>
+        <%= plucked_article[2] %> comments
```
Comment (rhymes) [inline app/views/articles/_widget_list_item.html.erb:8]: you could use this Rails trick: 

```ruby
<%= pluralize(plucked_article[2], 'comment') %>
```

which automatically decides if to write `comment` or `comments` if the number is above 1. 

see https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize

**Review by rhymes — APPROVED** [review]
Cool!

**Review by ludwiczakpawel — APPROVED** [review]
Yea, lets use that rails trick that @rhymes suggested because it gives cleaner code. Other than that it's a nice small improvement.

File: app/views/articles/_widget_list_item.html.erb (hunk 1 above)
Comment (benhalpern) [reply app/views/articles/_widget_list_item.html.erb:8]: @lisasy I'd definitely say making this adjustment would be worth it for code cleanliness. Change overall looks good though.

Comment (lisasy) [reply app/views/articles/_widget_list_item.html.erb]: @rhymes thank you, this is so helpful 😁

**Review by rhymes — APPROVED** [review]
LGTM!

**Pull request merged**